app/src/main/res/layout/activity_main.xml

```diff
@@ -18,4 +18,10 @@
         app:layout_constraintTop_toTopOf="parent"
         />
 
+    <ImageView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:paddingLeft="10dp"
```

Comment:
WARNING: [RtlSymmetry] When you define `paddingLeft` you should probably also define `paddingRight` for right-to-left symmetry
 If you specify padding or margin on the left side of a layout, you should probably also specify padding on the right side (and vice versa) for right-to-left layout symmetry.